src/test/suite/commandHandler.test.ts

```diff
@@ -52,6 +52,8 @@ suite("Command Handler tests", function () {
 		let changed = false;
 		if (DidactWebviewPanel.currentPanel) {
 			const panel = DidactWebviewPanel.currentPanel.getPanel();
+			expect(panel).to.not.equal(null);
+			expect(panel).to.not.equal(undefined);
```

Comment:
I think you can use expect(panel).to.exist

Reply:
Looks like that's not recommended - https://www.chaijs.com/api/bdd/#method_exist - but even trying what it recommends I'm seeing warnings

```
			expect(panel).to.not.be.null;
			expect(panel).to.not.be.undefined;
```

I'm getting "unused expression" warnings from tslint on these lines.